torch/_torch_docs.py

```diff
@@ -5376,6 +5376,28 @@ def merge_dicts(*dicts):
     tensor([ 1., -1.,  0.,  1.])
 """.format(**common_args))
 
+add_docstr(torch.sgn,
+           r"""
+sgn(input, out=None) -> Tensor
+
+Returns a new complex tensor with the same angle as that of the elements of :attr:`input` and absolute value 1.
+
+.. math::
+    \text{out}_{i} = \frac{{\text{{input}}_i}}{|{\text{{input}}_i}|}
+""" + r"""
+Args:
+    {input}
+    {out}
+
+.. warning:: :func:`sgn` is only supported for tensors with complex dtypes.
+
+Example::
+
+    >>> x=torch.randn(4, dtype=torch.cfloat)
```

Comment:
The example doesn't really demonstrate the function because of the random values. Maybe pick some angles, like 0, 45, 90 degrees?